bubble_sort.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+# rubocop:disable Metrics/AbcSize
+
+def bubble_sort(arr)
+  swap = true
+  while swap
+    swap = false
+    (arr.length - 1).times do |x|
+      swap = true if arr[x] > arr[x + 1]
+      arr[x], arr[x + 1] = arr[x + 1], arr[x] if arr[x] > arr[x + 1]
+    end
+  end
+  print arr
+end
+
+arr = [1, 4, 5, 66, 7, 3, 12_312, 2]
+
+bubble_sort(arr)
```

Comment:
- **[REQUIRED]** No debugging instructions in committed code (as discussed [here](https://questions.microverse.org/t/debugging-instructions-code/346)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/section/debugging_requirement.png) from Pathwright)